invenio_communities/views/communities.py

```diff
@@ -57,6 +61,27 @@
 ]
 
 
+def render_branded_template(template_name_or_list, theme_brand=None, **context):
+    """Render branded theme."""
+    if theme_brand:
+        if isinstance(template_name_or_list, str):
+            loader = BrandedThemeChoiceJinjaLoader(theme_brand)
+
+            branded_view_env = current_app.jinja_env.overlay(loader=loader)
+
+            template = branded_view_env.get_or_select_template(template_name_or_list)
+            app = current_app._get_current_object()
+            # not ideal using the private flask function
+            return _render(app, template, context)
```

Comment:
To check if there is a better way to handle this, very risky.
To mitigate, a [unit] test should be added, so the next version of flask removes this function, tests will start failing.

Reply:
done :)